src/Components/Comment.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { MDBRow, MDBCol, MDBIcon } from 'mdbreact';
+import './styles/Comment.css'
+
+
+const Comment = (props) => {
+    return (
+        <MDBRow className="comment-post">
+            <MDBCol size="1" className="comment-margin"></MDBCol>
+
+            <MDBCol size="1">
+                <img src="/default-avatar.jpg" className="comment-img"></img>
```

Comment:
Can we do dynamic avatars? They're supported on backend now. 

You can set up those users using postman or have a temp function during onboarding to randomly assign an avatar :) . OR have a function here which defaults to a certain avatar if none exists (this might be best for production environment robustness).